site/src/examples/low-barrel/index.js

```diff
@@ -206,6 +206,18 @@
                     ]);
                 }
                 return data;
+            })
+            .decorate(function(sel) {
+                var height = Math.abs(chart.yScale().range()[0] - chart.yScale().range()[1]);
+                sel.enter()
+                    .selectAll('.resize.e>rect, .resize.w>rect')
+                    .style('visibility', 'visible')
+                    .attr('y', height / 4)
+                    .attr('rx', 4)
+                    .attr('ry', 4);
+
+                sel.selectAll('.resize.e>rect, .resize.w>rect')
```

Comment:
Is there any reason why the height is set within the update selection, whereas the other properties of the `rect` are set in the enter selection?